src/utils/httpUtils.ts

```diff
@@ -1,6 +1,8 @@
+import { ORG_ID_REGEXP } from '../constants';
+
 export function getURL(baseURL: string, path: string, orgId?: string): string {
   if (routeToGateway(baseURL)) {
-    if (!orgId) {
+    if (!isValidOrg(orgId)) {
       throw new Error('Org is required for this operation');
```

Comment:
enh: The error message here may be a bit misleading - the user did provide an org id but the error implies it is missing.

Reply:
Fair enough, I corrected it to `A valid Org id is required`